.github/workflows/yarn.yaml

```diff
@@ -0,0 +1,30 @@
+name: yarn
+
+on:
+  - pull_request
```

Comment:
Should we only do this once a PR has merged?

```suggestion
  - pull_request
      types: [closed]
```

Reply:
Oh actually I see what this is doing. This will break Renovate as it will no longer rebase the change. Probably fine?

Reply:
Actually what I really want is to run this if any status in a Renovate PR build fails

Reply:
I could make it manually run based on a command like the Spotless one

Reply:
Probably need to start documenting these PR commands too 😉

Reply:
Switched